docs/src/examples/AccordionExample/AccordionExampleFirst.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Accordion, AccordionItemProps, DisplayText, Label } from '../../../../src/components/';
+import * as styles from '../../styles/components-page.scss';
+
+export interface Props {};
+
+export interface State {
+  Item1: boolean,
+  Item2: boolean,
+  Item3: boolean
+};
+
+class AccordionExample extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      Item1: false,
+      Item2: false,
+      Item3: false
+    };
+
+    this.Item1Toggle = this.Item1Toggle.bind(this);
+    this.Item2Toggle = this.Item2Toggle.bind(this);
+    this.Item3Toggle = this.Item3Toggle.bind(this);
+  }
+
+  private Item1Toggle() {
+    this.setState({
+      Item1: !this.state.Item1
+    });
+  }
+
+  private Item2Toggle() {
+    this.setState({
+      Item2: !this.state.Item2
+    });
+  }
+
+  private Item3Toggle() {
+    this.setState({
+      Item3: !this.state.Item3
+    });
+  }
+
```

Comment:
Where are you calling all the toggle functions, this looks wrong.

Reply:
Also is there a way, we can always open only 1 item, i.e. say item1 is open, and then when I click item2, item1 gets close and item2 gets open. We should have that functionality as well which can be driven via property, if anyone wants only 1 to be open at any given time.